### DISCUSSION TITLE
AUTO Waypoint Naming Conventions

### DISCUSSION THREAD

**Discussion by tina-hague** [Patribots4738/Crescendo2024#146] [Q&A]
HELLO COOL PEOPLE
What do we want the waypoint naming conventions to be in our auto paths? Possible ideas are saying...
- "Start" "Halfway" "End"
- "0" "1" "2"...
- actions -> "intake" "outtake" (if we did this how would we combine commands into one waypoint?)
What are your thoughts? :D

PLEASE WE NEED THIS TO FINISH AUTO (kind of)
PLEEEEEEEEEEEEEAAAAAASSSE

**Comment by HHQB** [comment]
For the start middle and end I was thinking about something like (1/3) for start, (2/3) middle, and (3/3) end. Probably not a finished idea but just a thought.

**Comment by GalexY727** [comment]
I think that since the markers glow when you hover them, you can have a name based on the action(s) that the marker performs. For example, if we wanted to intake and move the piece to the shooter, it could be named "intake to shooter" and if it was going to trap it would be "intake to trap"

**Reply by GalexY727** [reply]
another alternative would be "intake for shooter" and "intake for trap"

**Reply by jacueblol** [reply]
i like the English that this solution provides:DD

**Reply by tina-hague** [reply]
How do you suggest we would combine 2 commands into 1 waypoint? ie combining "PrepareShooterL" & "StopIntake" into 1 waypoint

**Reply by GalexY727** [reply]
You can just say 
PrepareL & stop intake
or just say PrepareL and stop intake would be assumed... not sure yet

**Comment by Oliver-Cushman** [comment]
I like the idea of it being more literal as @GalexY727 has already said, this would lead to minimal confusion.

**Comment by jacueblol** [comment]
This is resolved and is written down in the tldraw
